Server/Readme.md

```diff
@@ -13,10 +13,10 @@ Return to [main](./../Readme.md).
 
 For more details how to configure the plugins see the Readme in the project of the plugin. For the server the following settings are available:
 
-* <code>Delay</code> - time interval where the next keypress is checked
-* <code>PluginPath</code> - The path where the client is looking for datasource and datasink plugins, by default in <code>Plugins</code>
-* <code>PluginProductName</code> - Each plugin should have a specified product name to identify them as plugins for this software and to drop off the other DLL files. The default is <code>net.derpaul.tf.plugin</code> and this is valid __FOR ALL__ plugins.
-* <code>BrokerIP</code> - The IP of the system where the MQTT broker is running. The default is: <code>127.0.0.1</code>
+* <code>Delay</code> - time interval at which the next keypress is checked
+* <code>PluginPath</code> - The path where the client is looking for datasource and datasink plugins, by default in a subdirectory called <code>Plugins</code>
+* <code>PluginProductName</code> - Each plugin should have a specified product name to identify them as plugins for this software and to ignore non-plugin DLL files. The default is <code>net.derpaul.tf.plugin.[name]</code> and this is valid __FOR ALL__ plugins.
+* <code>BrokerIP</code> - The IP of the system where the MQTT broker is running. The default is: <code>localhost</code>

```

Comment:
`127.0.0.1` will be always available. You can modify the `localhost` entry in the host file to point to somewhere else.

Reply:
see the other comment for a reply